boards/arm/96b_argonkey/96b_argonkey.dts

```diff
@@ -62,7 +62,7 @@
 		compatible = "st,lsm6dsl";
 		reg = <1>;
 		spi-max-frequency = <1000000>;
-		irq-gpios = <&gpiob 1 0>;
+		irq-gpios = <&gpiob 1 GPIO_ACTIVE_HIGH>;
```

Comment:
I already addressed it in #21203. I think it is the proper place (96b_argonkey changes)

Reply:
But can be done also in this commit honestly.
What do you think?

Reply:
I don't care where it's done.  I'll leave it in here for now, there should be no conflict no matter which gets merged first.

Reply:
@avisconti Do you have objections to this PR other than that it updates the devicetree binding that it depends on?  I don't think I can remove that, unless #21203 is merged first and then this is rebased on it.

Reply:
> @avisconti Do you have objections to this PR other than that it updates the devicetree binding that it depends on? I don't think I can remove that, unless #21203 is merged first and then this is rebased on it.

Sure.